src/Trixi.jl

```diff
@@ -38,7 +38,7 @@ using ForwardDiff: ForwardDiff
 using HDF5: h5open, attributes
 using IfElse: ifelse
 using LinearMaps: LinearMap
-using LoopVectorization: LoopVectorization, @turbo, indices
+using LoopVectorization: LoopVectorization, @tturbo, @turbo, indices
```

Comment:
```suggestion
using LoopVectorization: LoopVectorization, @turbo, indices
```
I prefer using `@turbo thread=true` instead of `@tturbo` since that reduces the number of names and makes the purpose more clear, I think.